lib/sisimai/bite/email/gsuite.rb

```diff
@@ -0,0 +1,273 @@
+module Sisimai::Bite::Email
+  module GSuite
+    # Sisimai::Bite::Email::GSuite parses a bounce email which created by G Suite.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/GSuite.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
+        :from    => %r/[@]googlemail[.]com[>]?\z/,
+        :subject => %r/Delivery[ ]Status[ ]Notification/,
+      }
+      Re1 = {
+        :begin   => %r/\A[*][*][ ].+[ ][*][*]\z/,
+        :error   => %r/\AThe[ ]response([ ]from[ ]the[ ]remote[ ]server)?[ ]was:\z/,
+        :html    => %r{\AContent-Type:[ ]*text/html;[ ]*charset=['"]?(?:UTF|utf)[-]8['"]?\z},
+        :rfc822  => %r{\AContent-Type:[ ]*(?:message/rfc822|text/rfc822-headers)\z},
+        :endof   => %r/\A__END_OF_EMAIL_MESSAGE__\z/,
+      }
+      ErrorMayBe = {
+        :userunknown  => %r/because the address couldn't be found/,
+        :notaccept    => %r/Null MX/,
+        :networkerror => %r/DNS type .+ lookup of .+ responded with code NXDOMAIN/,
+      }
+      Indicators = Sisimai::Bite::Email.INDICATORS
+
+      def description; return 'G Suite: https://gsuite.google.com'; end
```

Comment:
__[rubocop]__
* Style/SingleLineMethods: Avoid single-line method definitions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-single-line-methods)
* Style/Semicolon: Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)
* Layout/TrailingWhitespace: Trailing whitespace detected. [:ref](https://github.com/bbatsov/ruby-style-guide#no-trailing-whitespace)

<!-- Comment by [SideCI](https://sideci.com) -->